virttest/utils_test/libvirt.py

```diff
@@ -2616,13 +2623,18 @@ def do_migration(vm_name, uri, extra, auth_pwd, auth_user="root",
             else:
                 logging.error("The real prompt text: <%s>", text)
                 break
-
+        log = session.get_output()
         session.close()
+        if return_ret_output is True:
+            return (True, log)
         return True
```

Comment:
I think you don't need add a new param to decide which kind of return you want, only update the function to return a tuple with status and log, then update all tp-libvirt cases with this update. If log not needed in some cases, just accepted it to _.